src/app/books.ts

```diff
@@ -2217,51 +2217,35 @@ export const BOOKS : Book[] = [
 		"id": 76,
 		"title": "Two Sides of Melody",
 		"edition": "1st",
-		"img": "https://cdn-img.fimfiction.net/story/xql3-1432435522-31474-large",
+		"img": "https://static.wixstatic.com/media/45d896_360adae6c0d0498f9a686bd0109ab92d~mv2_d_3965_2595_s_4_2.jpg/v1/fill/w_1125,h_717,al_c,q_85,usm_0.66_1.00_0.01/45d896_360adae6c0d0498f9a686bd0109ab92d~mv2_d_3965_2595_s_4_2.webp",
 		"rating": "T",
 		"dateAdded": "2019/01/06",
 		"expiry": null,
 		"authors":[
 			{
-				"name": "errant",
-				"url": "https://www.fimfiction.net/user/12443/errant"
-			},
-			{
-				"name": "Avensis Astari",
-				"url": "https://www.fimfiction.net/user/2168/Avensis+Astari"
-			},
-			{
-				"name": "Dennis the Menace",
-				"url": "https://www.fimfiction.net/user/6405/Dennis+the+Menace"
+				"name": "Ministry of Image",
+				"url": "https://www.ministryofimage.net"
 			}
 		],
 		"links": [
 			{
-				"url": "https://www.fimfiction.net/story/31474/accidental-harmony",
-				"title": "Fimfiction.net (Accidental Harmony)"
-			},
-			{
-				"url": "https://www.fimfiction.net/story/2365/allegrezza",
-				"title": "Fimfiction.net (Allegrezza)"
-			},
-			{
-				"url": "https://www.fimfiction.net/story/36388/my-roommate-is-a-vampire",
-				"title": "Fimfiction.net (My Roommate is a Vampire)"
+				"url": "https://www.fimfiction.net/bookshelf/1161649/two-sides-of-melody",
+				"title": "Fimfiction.net"
```

Comment:
I would REALLY much rather not point to bookshelves, but rather link to individual stories.